WPFHospitalEditor/ScheduleWindow.xaml.cs

```diff
@@ -57,7 +54,23 @@ private void scheduleBtn_Click(object sender, RoutedEventArgs e)
             }
         }
 
-        private void setAppointmentInfoContent()
+        private void PatientTextInputChanged(object sender, EventArgs e)
+        {
+            patientsComboBox.Items.Clear();
+            patientsComboBox.Items.Add("None");
+            patientsComboBox.SelectedIndex = 0;
+            SetPatientNameNameComboBox();
+        }
+
+        private void SetPatientNameNameComboBox()
+        {
+            foreach (Patient p in patientServerController.GetFilteredPatients(PatientSearchInput.Text))
+            {
+                patientsComboBox.Items.Add(p.Id.ToString() + " " + p.Person.Name + " " + p.Person.Surname + " - " + p.Person.Jmbg);
+            }
```

Comment:
You should add "None" option here and select index 0. Also you should change name of method to SetPatientNameComboBox.

Reply:
You are right, I have fixed it.